scootapi/locate.go

```diff
@@ -24,6 +24,9 @@ func GetScootapiAddrPath() string {
 func GetScootapiAddr() (sched string, api string, err error) {
 	data, err := ioutil.ReadFile(GetScootapiAddrPath())
 	if err != nil {
+		if os.IsNotExist(err) {
+			return "", "", nil
```

Comment:
The same way ReadFile returns an error, but not EOF, this shouldn't error if the file doesn't exist, because it's trying to read there. It *should* error if, I dunno, it fails to read because your hard drive died in the middle.